Mathlib/Data/ZMod/Coprime.lean

```diff
@@ -37,4 +37,114 @@ theorem eq_zero_of_gcd_ne_one {a : ℤ} {p : ℕ} (pp : p.Prime) (h : a.gcd p 
   (@eq_zero_iff_gcd_ne_one a p ⟨pp⟩).mpr h
 #align zmod.eq_zero_of_gcd_ne_one ZMod.eq_zero_of_gcd_ne_one
 
+open Nat
```

Comment:
All of the results below this line end up in the `ZMod` namespace, but none of them are about `ZMod`! I think they belong somewhere else.

Reply:
Moved theorem related to `chineseRemainderList` (remaned to `chineseRemainderOfList`) to Data.Nat.ModEq.